index.html

```diff
@@ -1573,10 +1566,10 @@ <h2 id="vp-ld-json-jwt"><code>application/vp+ld+json+jwt</code></h2>
         </section>
 
         <section id="vc-ld-sd-jwt-media-type">
-            <h2 id="vc-ld-json-sd-jwt"><code>application/vc+ld+json+sd-jwt</code></h2>
+            <h2 id="vc-ld-json-sd-jwt"><code>application/vc+sd-jwt</code></h2>
             <p>
                 This specification registers the
-                <code>application/vc+ld+json+sd-jwt</code> Media Type specifically for
+                <code>application/vc+sd-jwt</code> Media Type specifically for
```

Comment:
This registration is already requested in https://datatracker.ietf.org/doc/html/draft-ietf-oauth-sd-jwt-vc-03#name-application-vcsd-jwt

Reply:
Pedantically speaking the request has not yet been made but there is intent to do so. https://mailarchive.ietf.org/arch/msg/media-types/2VdBm8c-GVcYf3GrWGyQSkJ-NVM/